### DISCUSSION TITLE
starstruck badge

### DISCUSSION THREAD

**Discussion by ashishkumar90052** [drknzz/GitHub-Achievements#40] [Q&A]
Need star for starstruck badge.
Give star to get badge.

https://github.com/ashishkumar90052/Instagram-Clone

**Comment by DanielR-spec** [comment]
Thank u for ur support :100:

**Comment by nguyen-tuan-hiep** [comment]
GIve me a star on my repo: https://github.com/nguyen-tuan-hiep/web-project and I will give you back
